eventstore/models.py

```diff
@@ -197,6 +197,22 @@ class PrebirthRegistration(models.Model):
     data = JSONField(default=dict, blank=True, null=True)
 
 
+class PMTCTRegistration(models.Model):
+    normal = "Normal"
+    high = "High"
```

Comment:
It's django convention to have the variable names uppercase, as they're considered magic numbers. What we want is the string values to be lowercase, e.g.
```
NORMAL = "normal"
RISK_TYPES = ((NORMAL, "Normal"),)
```
That will give a machine value of `normal`, and a human value of `Normal`